### PR TITLE
Strings

### PR DESCRIPTION
**Pull request by mahdiHashemi14**
Please review and comment/commit if anything is wrong

### CODE REVIEW

File: 1-js/05-data-types/03-string/4-extract-currency/task.md [hunk 1]
```diff
@@ -2,13 +2,13 @@ importance: 4
 
 ---
 
-# Extract the money
+# استخراج پول
 
-We have a cost in the form `"$120"`. That is: the dollar sign goes first, and then the number.
+ما یک قیمت در شکل `"$120"` داریم. به این معنی که علامت دلار اول می‌آید، و سپس عدد.
```
Comment (mahdyar) [inline 1-js/05-data-types/03-string/4-extract-currency/task.md:7]:
```suggestion
ما یک قیمت به شکل `"$120"` داریم. به این معنی که علامت دلار اول می‌آید، و سپس عدد.
```

File: 1-js/05-data-types/03-string/article.md [hunk 2]
```diff
@@ -27,246 +27,246 @@ function sum(a, b) {
...
 ```js run
 let str = `Hello`;
 
 alert( str[1000] ); // undefined
-alert( str.charAt(1000) ); // '' (an empty string)
+alert( str.charAt(1000) ); // '' (یک رشته خالی)
 ```
 
-We can also iterate over characters using `for..of`:
+همچنین ما می‌توانیم با استفاده از `for..of` در رشته حلقه بزنیم:
```
Comment (mahdyar) [inline 1-js/05-data-types/03-string/article.md:171]:
```suggestion
همچنین ما می‌توانیم با استفاده از `for..of` برای کاراکترها حلقه بزنیم:
```

File: 1-js/05-data-types/03-string/article.md [hunk 3]
```diff
@@ -27,246 +27,246 @@ function sum(a, b) {
...
 
 ```js run
 for (let char of "Hello") {
-  alert(char); // H,e,l,l,o (char becomes "H", then "e", then "l" etc)
+  alert(char); // H، e، l، l، o، (و غیره "l" سپس ،"e" سپس ،"H" می‌شود char)
```
Comment (mahdyar) [inline 1-js/05-data-types/03-string/article.md:175]:
```suggestion
  alert(char); // H,e,l,l,o (و غیره "l" سپس ،"e" سپس ،"H" می‌شود char)
```

File: 1-js/05-data-types/03-string/article.md [hunk 4]
```diff
@@ -27,246 +27,246 @@ function sum(a, b) {
...
 ```
 
-## Strings are immutable
+## رشته‌ها تغییرناپذیر هستند
 
-Strings can't be changed in JavaScript. It is impossible to change a character.
+رشته‌ها در جاوااسکریپت نمی‌توانند تغییر کنند. اینکه یک کاراکتر را تغییر دهیم غیر ممکن است.
 
-Let's try it to show that it doesn't work:
+بیایید برای نشان دادن اینکه کار نخواهد کرد امتحانش کنیم:
```
Comment (mahdyar) [inline 1-js/05-data-types/03-string/article.md:183]:
```suggestion
بیایید برای نشان دادن اینکه این کار نخواهد کرد امتحانش کنیم:
```

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Comment by mahdiHashemi14** [comment]
Thank you for your review @mahdyar
/done

**Review by mahdyar — APPROVED** [review]
Thank you, @mahdiHashemi14!

**Pull request merged**